src/bulk-load.ts

```diff
@@ -508,6 +515,21 @@ class BulkLoad extends EventEmitter {
       addOptions.push('TABLOCK');
     }
 
+    if (this.bulkOptions.order.length) {
+      const orderColumns = [];
+
+      for (const column of this.bulkOptions.order) {
+        // Ensure column exists as a specified column, otherwise we ignore.
+        if (column.columnName && this.columnsByName[column.columnName] !== null) {
```

Comment:
I think if `column.columnName` is anything but a string, we should maybe throw an exception? 🤔 

Also, I don't think the strict null check here is required/correct? If the column is not set, `this.columnsByName[column.columnName]` should return `undefined` and not `null`.

Reply:
I now check `order` to be an 'object' and `columnName` to be a 'string', and throw TypeError if they aren't. How does it look?